src/tools/compiletest/src/main.rs

```diff
@@ -452,11 +452,6 @@ pub fn run_tests(config: &Config) {
         _ => { /* proceed */ }
     }
 
-    // FIXME(#33435) Avoid spurious failures in codegen-units/partitioning tests.
```

Comment:
This is the only removal that gives me pause, the rest LGTM.

Reply:
Checked the issue again. This was the fix mentioned, so reverting commit.